docs/sisl.md

```diff
@@ -2,11 +2,15 @@
 
 {% include bigtable.html group="sisl/" avg_rew=1 %}
 
-`pip install pettingzoo[sisl]`
```

Comment:
Please revert all changes to this file, I'll fix it myself after everything is merged.

Reply:
Sure thing